packages/manager/apps/dedicated/client/app/account/user/users/sso/delete/user-users-sso-delete.html

```diff
@@ -0,0 +1,15 @@
+<div data-ng-controller="UserAccountUsersSSODeleteCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="deleteSSO"
```

Comment:
I guess better like this:
```suggestion
        data-wizard-on-finish="deleteSso"
```